docs/shared/revisions.txt

```diff
@@ -26,6 +26,7 @@
 [cols="2",options="header"]
 |===
 | Version    | Date
+| 2.1.0      | TBD
 | 2.0.1      | July 7, 2016
 | 2.0.0      | June 6, 2016
 | 1.3.0      | January, 2016  
```

Comment:
Yes, it is shared in all documents. My thought was that perhaps the release manager could supply the date at the tail end of the release cycle. If this is agreeable, I can add a comment to the wiki as you suggest.